src/Chainweb/Pact/TransactionExec.hs

```diff
@@ -519,7 +529,7 @@ readInitModules logger dbEnv txCtx
       refModsCmd <- liftIO $ mkCmd $ T.intercalate " " $
         [ "coin.MINIMUM_PRECISION"
         , "ns.GUARD_SUCCESS"
-        , "gas-payer-v1.GAS_PAYER"
+        , "(use gas-payer-v1)"
```

Comment:
note this could be left as-is and we could instead fork pre 219 and post 219, but this provides a compatible solution which doesn't seem to impact blocks (pending replay verification).

Reply:
I'm in favor of reverting until we can prove that this summons the modules in the correct way. It should, but this is a small change in an orthogonal PR, and it would be better to test on its own as a PR.

Reply:
It is not orthogonal. It does not pass replay without this change.

Reply:
Also, this passed replay.

Reply:
Ah, okay great